gapir/cc/grpc_replay_service.cpp

```diff
@@ -95,6 +95,27 @@ std::unique_ptr<ReplayService::Payload> GrpcReplayService::getPayload(
       std::unique_ptr<replay_service::Payload>(req->release_payload())));
 }
 
+std::unique_ptr<ReplayService::ProfilerReady>
+GrpcReplayService::getProfilerReady(const uint64_t& id) {
+  // Send a replay response with payload request
+  replay_service::ReplayResponse res;
+  auto prc = new replay_service::ProfilerReadyRequest();
+  prc->set_id(id);
+  res.set_allocated_profiler_ready_request(prc);
+  mGrpcStream->Write(res);
+  std::unique_ptr<replay_service::ReplayRequest> req = getNonReplayRequest();
```

Comment:
I see this is a pattern we've used before, but here this is getting a bit tricky in that, once a wait ready event is sent, we can only receive a wait ready response. For now, I think that's OK, but it's definitely something to keep in mind.

Reply:
@apbodnar file a bug for fixing this properly?

Reply:
I've created the bug:  b/143093342